osfoffline/views/rsc/preferences_rc.py

```diff
@@ -107,14 +110,15 @@ def retranslateUi(self, Preferences):
         self.changeFolderButton_2.setText(_translate("Preferences", "Update"))
         self.treeWidget.headerItem().setText(0, _translate("Preferences", "Projects"))
         self.treeWidget.headerItem().setText(1, _translate("Preferences", "Sync"))
+        self.pushButton.setText(_translate("Preferences", "Sync All"))
```

Comment:
Would it be a major hassle to give your items more descriptive names? (similar to how other buttons are named- `changeFolderButton`, `accountLogOutButton`, etc)

That would make it easier to maintain code that connects to these buttons.